torch/onnx/symbolic_opset13.py

```diff
@@ -109,6 +109,136 @@ def unsafe_split_with_sizes(g, self, split_sizes, dim, _outputs=None):
     return split_with_sizes(g, self, split_sizes, dim, _outputs)
 
 
+@symbolic_helper.parse_args("v", "v", "i", "i")
+def tensor_split(g, self, indices_or_sections, dim, _outputs=None):
+    axis = g.op("Constant", value_t=torch.tensor(dim, dtype=torch.long))
+    axis = opset11.unsqueeze(g, axis, 0)
+
+    def static_tensor_split(self, indices):
+        start = g.op("Constant", value_t=torch.tensor([0], dtype=torch.long))
+        res = []
+        for i in range(_outputs - 1):
+            end = g.op("Gather", indices_or_sections, i, axis)
+            res.append(g.op("Slice", self, start, end, axis))
+            start = end
+
+        end = symbolic_helper._size_helper(g, self, axis)
+        res.append(g.op("Slice", self, start, end, axis))
+        return res
+
+    if not symbolic_helper._is_split_static(indices_or_sections, _outputs):
```

Comment:
Since in most scenarios the dynamic code will be triggered, would it be better to put the dynamic code in front of the static one?

The conversion of `torch.split` in this file adopts the same approach.

Reply:
Well if we can't find any input that triggers the static case, I think we should remove the static case altogether.
But if we do keep both, the organization of the code should be optimized for readability, not based on how often we guess each branch of the if will be executed.

Reply:
OK, makes sense.